python/gink/impl/lmdb_store.py

```diff
@@ -314,11 +315,6 @@ def get_reset_changes(self, to_time: MuTimestamp, container: Optional[Muid],
                     for change in self._container_reset_changes(to_time, muid, seen, txn):
                         yield change
                     cursor_placed = containers_cursor.next()
-                # then loop over the "magic" pre-defined
-                for behavior in [DIRECTORY, SEQUENCE, BOX, KEY_SET]:
-                    muid = Muid(-1, -1, behavior)
-                    for change in self._container_reset_changes(to_time, muid, seen, txn):
-                        yield change
```

Comment:
The global containers are already looped over in the first container_reset_changes call, so this was causing duplicate resets.

I'm not sure what I did for this to just now start failing the test, but restoring a previously reset global sequence was appending the same value twice.